cpp/tests/test_abm_model.cpp

```diff
@@ -789,6 +791,11 @@ TEST_F(TestModel, checkParameterConstraints)
     EXPECT_TRUE(params.check_constraints());
     params.get<mio::abm::MaskProtection>()[mio::abm::MaskType::Surgical] = 0.7;
 
+    params.get<mio::abm::QuarantineEffectiveness>() = -0.1;
+    EXPECT_TRUE(params.check_constraints());
+    params.get<mio::abm::QuarantineEffectiveness>() = 1.5;
+    EXPECT_TRUE(params.check_constraints());
+
```

Comment:
Should be set to a valid value after the checks.

Reply:
Good catch